release.md

```diff
@@ -82,6 +82,16 @@ After the release pull request has been merged into the `main` branch, it is tim
 
 ## Release on conda-forge
 
+In order to release on conda-forge, you can either wait for a bot to create a PR, or manually kickoff the creation with GitHub Actions
+
+### Option 1: Manually create the new PR with GitHub Actions
+1. Go to this GitHub Action: https://github.com/alteryx/featuretools/actions/workflows/create_feedstock_pr.yaml
+2. Input the released version with the v prefix (e.g. v0.13.3)
+3. Kickoff the GitHub action, and monitor the Job Summary. At the completion of the job, you should see summary output, with the URL: 
+  a. You can also just go to: https://github.com/machineAYX/featuretools-feedstock/pull/new/conda-autocreate-v0.13.3 (change the last part to the released version)
```

Comment:
I think we might want to specify that the user also needs to visit the URL and submit the PR

Reply:
Fixed
https://github.com/alteryx/featuretools/pull/2181/commits/854e1b1b31d8da2469eb53a3b233e7ccbcc44df3